src/vivarium_gates_mncnh/model_specifications/model_spec.yaml

```diff
@@ -81,6 +81,44 @@ configuration:
     stratification:
         default:
             - 'age_group'
```

Comment:
You're excluding `age_group` from every observation, so why is it the default? Wouldn't `child_age_group` be a better default? Depending on how you're feeling, `child_sex` and/or `delivery_facility_choice` might make sense to be in the defaults as well.

Reply:
I have updated this to not use defaults. Defaults just do not work for all of the observers because of the wide state table.